libs/community/langchain_community/graph_vectorstores/links.py

```diff
@@ -9,10 +9,113 @@
 @beta()
 @dataclass(frozen=True)
 class Link:
-    """A link to/from a tag of a given tag.
+    """A link to/from a tag of a given kind.
 
-    Edges exist from nodes with an outgoing link to nodes with a matching incoming link.
-    """
+    Documents in a :class:`graph vector store <langchain_community.graph_vectorstores.base.GraphVectorStore>`
+    are connected via "links".
+    Links form a bipartite graph between documents and tags: documents are connected
+    to tags, and tags are connected to other documents.
+    When documents are retrieved from a graph vector store, a pair of documents are
+    connected with a depth of one if both documents are connected to the same tag.
+
+    Links have a ``kind`` property, used to namespace different tag identifiers.
+    For example a link to a keyword might use kind ``kw``, while a link to a URL might
+    use kind ``url``.
+    This allows the same tag value to be used in different contexts without causing
+    name collisions.
+
+    Links are directed. The directionality of links controls how the graph is
+    traversed at retrieval time.
+    For example, given documents ``A`` and ``B``, connected by links to tag ``T``:
+
+    +----------+----------+---------------------------------+
+    | A to T   | B to T   | Result                          |
+    +==========+==========+=================================+
+    | outgoing | incoming | Retrieval traverses from A to B |
+    +----------+----------+---------------------------------+
+    | incoming | incoming | No traversal from A to B        |
+    +----------+----------+---------------------------------+
+    | outgoing | incoming | No traversal from A to B        |
+    +----------+----------+---------------------------------+
+    | bidir    | incoming | Retrieval traverses from A to B |
+    +----------+----------+---------------------------------+
+    | bidir    | outgoing | No traversal from A to B        |
+    +----------+----------+---------------------------------+
+    | outgoing | bidir    | Retrieval traverses from A to B |
+    +----------+----------+---------------------------------+
+    | incoming | bidir    | No traversal from A to B        |
+    +----------+----------+---------------------------------+
+
+    Directed links make it possible to describe relationships such as term
+    references / definitions: term definitions are generally relevant to any documents
+    that use the term, but the full set of documents using a term generally aren't
+    relevant to the term's definition.
+
+    .. seealso::
+
+        - :mod:`How to use a graph vector store <langchain_community.graph_vectorstores>`
```

Comment:
cool -- curious to see how well all of these will render on the APIReference page!

Reply:
You can see now: https://python.langchain.com/api_reference/community/graph_vectorstores/langchain_community.graph_vectorstores.links.Link.html 😉